deploy/kubeturbo_yamls/step5_turbo_kubeturboDeploy.yaml

```diff
@@ -30,6 +30,7 @@ spec:
           # change to https=false and port=10255 if unsecure kubelet read only is configured
           - --kubelet-https=true
           - --kubelet-port=10250
+          - --mergeactions=false
```

Comment:
Maybe remove this?

Reply:
Maybe this is better:

Reply:
This change is only against 7.22.5 branch, so it should be ok to leave it there.